### PR TITLE
Group environments by type in docs

### PR DESCRIPTION
**Pull request by LucasAlegre**
As suggested in #82.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -10,11 +10,21 @@ lastpage:
...
 ```
 
+```{toctree}
+:hidden:
+:caption: Environments
+
+environments/discrete-state
+environments/continuous-state
+environments/image-state
+environments/mujoco
```
Comment (ffelten) [inline docs/index.md:24]: Looks weird to have mujocos under continuous state AND mujoco. I see two ways of separating:
## By family
Mujocos, grid worlds (~all discrete), classical (mountain car, lunar lander), misc (minecart, Mario, highway)

## By obs spaces
Like you did here, but removing Mujocos

**Review by ffelten — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (LucasAlegre) [reply docs/index.md:24]: do you have a preference?

Comment (ffelten) [reply docs/index.md:24]: Family, as in Gym?

Comment (LucasAlegre) [reply docs/index.md:24]: It is better indeed

**Pull request merged**